app/src/main/java/com/example/ar_reshare/SettingActivity.java

```diff
@@ -0,0 +1,251 @@
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+
+                    }
+                });
+                dialog2.show();
+            }
+        });
+
+        Button birthdayButton = (Button) findViewById(R.id.birthday);
+        birthdayButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                AlertDialog.Builder dialog3 = new AlertDialog.Builder(v.getContext());
+                dialog3.setIcon(R.drawable.settings_birthday);
+                dialog3.setTitle("Please insert your new birthday");
+
+                View view = LayoutInflater.from(v.getContext()).inflate(R.layout.brithday_dialog, null);
+                dialog3.setView(view);
+
+                final EditText Year = (EditText) view.findViewById(R.id.Year);
+                final EditText Month = (EditText) view.findViewById(R.id.Month);
+                final EditText Day = (EditText) view.findViewById(R.id.Day);
+
+                dialog3.setPositiveButton("Yes", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        String a = Year.getText().toString().trim();
+                        String b = Month.getText().toString().trim();
+                        String c = Day.getText().toString().trim();
+
+                        Toast.makeText(v.getContext(), "New Birthday:" + a +"_"+ b + "_" + c, Toast.LENGTH_SHORT).show();
+                    }
+                });
+                dialog3.setNegativeButton("No", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+
+                    }
+                });
+                dialog3.show();
+            }
+        });
+
+        Button pwdButton = (Button) findViewById(R.id.pwd);
+        pwdButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                AlertDialog.Builder dialog4 = new AlertDialog.Builder(v.getContext());
+                dialog4.setIcon(R.drawable.setting_pwd);
+                dialog4.setTitle("Please insert your new Password");
+
+                View view = LayoutInflater.from(v.getContext()).inflate(R.layout.pwd_dialog, null);
+                dialog4.setView(view);
+
+                final EditText pwd = (EditText) view.findViewById(R.id.pwd);
+
+                dialog4.setPositiveButton("Yes", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        String a = pwd.getText().toString().trim();
+
+                        Toast.makeText(v.getContext(), "New password:" + a, Toast.LENGTH_SHORT).show();
```

Comment:
Could you please change the toast to just show "Your password has now been changed"? Instead of displaying the password back